<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_screen</class>
 <widget class="QWidget" name="Game_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>562</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_Game" native="true">
     <property name="minimumSize">
      <size>
       <width>562</width>
       <height>269</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>562</width>
       <height>269</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_Game{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.005013 rgba(42, 180, 255, 255), stop:0.279732 rgba(70, 222, 255, 255), stop:0.517588 rgba(110, 225, 255, 255), stop:0.768844 rgba(81, 255, 235, 255), stop:1 rgba(28, 249, 255, 255));
}

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_RegresiveCount" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lbl_numbers{
color: rgb(255, 85, 127);
font-family: verdana;
font-size:55px;
font-weight: bold;
font-style: italic;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
          <widget class="QLabel" name="lbl_numbers">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Questions" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background:transparent;
}

QWidget{
	background:transparent;
}

QLineEdit {
    background-color: white;
    border: 1px solid gray;
    border-radius: 4px;
    color: black;
}

QLineEdit:hover {
    background-color: rgb(213, 213, 213);
    border: 1px solid blue;
    color: black;
}

QLineEdit:!focus {
    color: black;
}

QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.005013 rgba(42, 180, 255, 255), stop:0.279732 rgba(70, 222, 255, 255), stop:0.517588 rgba(110, 225, 255, 255), stop:0.768844 rgba(81, 255, 235, 255), stop:1 rgba(28, 249, 255, 255));
border-style:inset;
border-width: 2px;
border-color: rgb(85, 170, 255);
border-radius: 3px;
color: black;
font-weight: bold;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(57, 112, 255, 255));
border-color: rgb(0, 85, 255);
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(57, 255, 108, 255));
border-color: rgb(0, 255, 127);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="lbl_Question">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1000</width>
             <height>1000</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
             <widget class="QLineEdit" name="txt_Question">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>22</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::LeftToRight</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
             <widget class="QPushButton" name="btn_Question">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>Responder</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="lbl_MsgAnswer">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Results" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	background:transparent;
}

QWidget{
	background:transparent;
}

QLineEdit {
    background-color: white;
    border: 1px solid gray;
    border-radius: 4px;
    color: black;
}

QLineEdit:hover {
    background-color: rgb(213, 213, 213);
    border: 1px solid blue;
    color: black;
}

QLineEdit:!focus {
    color: black;
}

QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.005013 rgba(42, 180, 255, 255), stop:0.279732 rgba(70, 222, 255, 255), stop:0.517588 rgba(110, 225, 255, 255), stop:0.768844 rgba(81, 255, 235, 255), stop:1 rgba(28, 249, 255, 255));
border-style:inset;
border-width: 2px;
border-color: rgb(85, 170, 255);
border-radius: 3px;
color: black;
font-weight: bold;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(57, 112, 255, 255));
border-color: rgb(0, 85, 255);
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(57, 255, 108, 255));
border-color: rgb(0, 255, 127);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
             <widget class="QLabel" name="lbl_ScoreMsg">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>14</pointsize>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
color: black;
}</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
             <widget class="QPushButton" name="lbl_FinishGame">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
